OLM_V1_MIGRATION_PROGRESS.md

```
@@ -0,0 +1,221 @@
+|--------|--------|--------|
+| Install Resource | Subscription (namespaced) | ClusterExtension (cluster-scoped) |
+| Catalog Resource | CatalogSource (namespaced) | ClusterCatalog (cluster-scoped) |
+| Namespace | Operator decides | ClusterExtension.spec.installNamespace |
+| OperatorGroup | Required | Not used |
+| Channel Selection | Subscription.spec.channel | ClusterExtension.spec.channel |
+| Version Pinning | Subscription.spec.startingCSV | ClusterExtension.spec.version |
+| Approval | Subscription.spec.installPlanApproval | ClusterExtension.spec.install.preflight.crdUpgradeSafety |
+
+### Import Cycle Issue
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)